core/src/main/scala/com/salesforce/op/OpWorkflowModelWriter.scala

```diff
@@ -165,9 +165,9 @@ private[op] object OpWorkflowModelReadWriteShared {
     val values = findValues
     case object Uid extends FieldNames("uid")
     case object ResultFeaturesUids extends FieldNames("resultFeaturesUids")
-    case object BlacklistedFeaturesUids extends FieldNames("blacklistedFeaturesUids")
-    case object BlacklistedMapKeys extends FieldNames("blacklistedMapKeys")
-    case object BlacklistedStages extends FieldNames("blacklistedStages")
+    case object BlocklistedFeaturesUids extends FieldNames("blocklistedFeaturesUids")
```

Comment:
This change would any existing models with a blacklist defined behave incorrectly. I would recommend to keep backwards compatibility of `resolveBlocklist` (fka `resolveBlacklist`) at least until the next major release, i.e. 0.8.0.

Reply:
Good point. Let me try to keep backwards compatibility then. Marking as WIP for now.

Reply:
I've added backwards compatibility and a test for it, @tovbinm